automation/az_elev_chart.py

```diff
@@ -0,0 +1,210 @@
+import datetime
```

Comment:
az_elev_chart.py has some problems:

1. if it's expected to be run by the user, then why is the not `usr/bin/env python` in the first line?
2. there's no help line at all;
3. I've ran this tool without any parameters. After couple seconds, it simply returned. I'd like

Reply:
This script shouldn't be directly executed by user.
You should use CLI:
    ```python cli.py pass PASS```
where PASS is pass number from ```python cli.py plan``` or satellite name.
If you provide satellite name then you may provide also ```--aos DATE``` where date is date in ISO8601 (UTC only). If you don't provide it then next pass will be drawn.